src/app/counters/counters.page.html

```diff
@@ -0,0 +1,86 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>
+      Counters
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <ion-list #listCounters>
+    <ion-item-sliding *ngFor="let counter of counters">
+      <ion-item>
+
+        <ng-template [ngIf]="counter.minusCount < 0">
+          <ion-button
+            size="large"
+            color="danger"
+            (click)="addInstance(counter, counter.minusCount)"
+          >
+          <div [ngSwitch]="counter.minusCount">
+            <div *ngSwitchCase="-1">
+              <ion-icon name="add"></ion-icon>
+            </div>
+
+            <div *ngSwitchDefault>
+              {{counter.minusCount}}
+            </div>
+
+          </div>
+          </ion-button>
+        </ng-template>
+
+        <ion-label class="ion-text-center">
+          <h1>{{counter.title}}</h1>
+          <h1>{{counter.count}}</h1>
+        </ion-label>
+
+        <ng-template [ngIf]="counter.plusCount > 0">
+          <ion-button
+            size="large"
+            color="success"
+            (click)="addInstance(counter, counter.plusCount)"
+          >
+          <div [ngSwitch]="counter.plusCount">
+            <div *ngSwitchCase="1">
+              <ion-icon name="add"></ion-icon>
+            </div>
+
+            <div *ngSwitchDefault>
+              +{{counter.plusCount}}
+            </div>
+
+          </div>
+          </ion-button>
+        </ng-template>
+
+      </ion-item>
+
+      <ion-item-options side="end" appNotImplemented description="Edit instances" issueId="8" on="ionSwipe">
+        <ion-item-option color="danger">
+          <ion-icon slot="icon-only" name="trash" (click)="deleteCounter(counter)"></ion-icon>
+        </ion-item-option>
+
+        <ion-item-option color="warning" >
+          <ion-icon slot="icon-only" name="build"></ion-icon>
+        </ion-item-option>
+
+        <ion-item-option color="success" appNotImplemented description="Show basic analytics" issueId="6">
+          <ion-icon slot="icon-only" name="analytics"></ion-icon>
+        </ion-item-option>
+
+        <ion-item-option expandable appNotImplemented description="Edit instances" issueId="8">
+          <ion-icon slot="icon-only" name="create" expandable></ion-icon>
+        </ion-item-option>
+
+      </ion-item-options>
+
+    </ion-item-sliding>
+  </ion-list>
+  <ion-button size="large" expand="full" (click)="addCounter()">
+    <ion-icon name="add"></ion-icon>
+  </ion-button>
+</ion-content>
```

Comment:
Da chame sicher de no einzelni Komponente drus mache.

Reply:
jop. Hani sogar o mau so gha, u e `Counter` komponaente agleit gha. Dr einizig grund wuerum is momentan no nid gmacht ha isch nume wueu mir dr name `Counter` nid passt. Finge mir hei langsam chli viu `Counters`, `Counter`, `CountInstance` ... was villech druf hiiduetet das mirs mit dr zerlegig chli zwit tribe?

Reply:
done. see `PlusMinusCounterComponent`